openapi.yaml

```diff
@@ -5693,7 +5693,7 @@
             "type" : "string"
           }
         },
-        "required" : [ "connectorConfig", "connectorName", "description", "environment" ]
```

Comment:
I don't think we should change that here, since the openapi file is usually autogenerated in the backend. We should keep that in sync, so openapi.yaml is only changed due backend changes and the types for frontend are then generated from it.

Reply:
Hmm, i see, thanks! i may have misunderstood the source of truth concept here.

Reply:
There are still changes there, so currently can't approve 😅